src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="big">
+      <form
+        action=""
```

Comment:
```suggestion
        action="#"
```
Не оставляй action пустым